### ISSUE TITLE
Images need alt text

### ISSUE THREAD

**Issue by cehfisher** [lauriii/umami#142]
I assume there is no alt text for the current images because they are just placeholders, but opening this ticket as a reminder to update this when the real images are in place.

**Comment by DuaelFr** [comment]
The problem is that there is no alt attribute at all. Even decorative images should at least have an empty alt attribute. :)

**Comment by markconroy** [comment]
The two image fields currently on content types both have alt field enabled and set as required. So, I presume this will be fixed before 8.5.x
![screen shot 2018-01-11 at 20 13 14](https://user-images.githubusercontent.com/2183332/34845222-10193032-f70c-11e7-9247-7944e40a22ba.png)

**Comment by fuzzbomb** [comment]
The custom banner block type doesn't have the imagefield alt text enabled.  Found that during Tuesday's review.

**Comment by fuzzbomb** [comment]
Is Umami theme the right place for this issue? Field config and CSV content is the profile repo....

**Comment by markconroy** [comment]
I came here to say the same thing! I'll close this one.
